packages/pds/src/db/migrations/20230118T223059239Z-repo-sync-data.ts

```diff
@@ -2,24 +2,33 @@ import { chunkArray } from '@atproto/common'
 import { MemoryBlockstore } from '@atproto/repo'
 import { Kysely } from 'kysely'
 import { CID } from 'multiformats/cid'
-import DatabaseSchema from '../database-schema'
 import { RepoCommitBlock } from '../tables/repo-commit-block'
 import { RepoCommitHistory } from '../tables/repo-commit-history'
 
 const commitBlockTable = 'repo_commit_block'
 const commitHistoryTable = 'repo_commit_history'
 
-export async function up(db: DatabaseSchema): Promise<void> {
+export async function up(db: any): Promise<void> {
```

Comment:
Could we use `Kysely<unknown>` here as we do in some of the other migrations?

Reply:
yup good call.

altho i fixed it in the sync revamp branch instead: https://github.com/bluesky-social/atproto/pull/446/commits/dcbad25f691d46c0ea2e683edbceb86ff74f7ae9